nx/lib/nx.ex

```diff
@@ -71,7 +71,7 @@ defmodule Nx do
         [2, 4, 6]
       >
 
-  In practice, brodcasting is not restricted only to scalars, it
+  In practice, broadcasting is not restricted only to scalars. It
```

Comment:
```suggestion
  In practice, broadcasting is not restricted only to scalars; it
```

I think this might read better

Reply:
I agree.